nixos/modules/services/networking/wireguard.nix

```diff
@@ -436,7 +436,13 @@ in
     networking.wireguard = {
 
       enable = mkOption {
-        description = lib.mdDoc "Whether to enable WireGuard.";
+        description = lib.mdDoc ''
```

Comment:
Do we already pull in the module package for old kernels when networkd config for wireguard is present? If not, that's something we should maybe think about. Though I'm not sure we have many people using such old kernels?

Reply:
Also the inclusion of wireguard-tools in systemPackages (especially for key generation) could similarly make sense.